subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/CacheLayout.java

```diff
@@ -15,49 +15,72 @@
  */
 package org.gradle.api.internal.artifacts.ivyservice;
 
-import org.gradle.util.VersionNumber;
+import org.gradle.cache.internal.CacheVersion;
+import org.gradle.cache.internal.CacheVersionMapping;
 
 import java.io.File;
 
+import static org.gradle.cache.internal.CacheVersionMapping.introducedIn;
+
 public enum CacheLayout {
-    ROOT(null, "modules", 2),
-    FILE_STORE(ROOT, "files", 1),
-    META_DATA(ROOT, "metadata", 58),
-    RESOURCES(ROOT, "resources", 1),
-    TRANSFORMS(null, "transforms", 1),
-    TRANSFORMS_META_DATA(TRANSFORMS, "metadata", 1),
-    TRANSFORMS_STORE(TRANSFORMS, "files", 1);
-
-    // If you update the META_DATA version, also update DefaultGradleDistribution.getArtifactCacheLayoutVersion() (which is the historical record)
+
+    ROOT(null, "modules", introducedIn("1.9-rc-1").incrementedIn("1.9-rc-2")),
+
     // If you update FILE_STORE, you may also need to update LocallyAvailableResourceFinderFactory
+    FILE_STORE(ROOT, "files", introducedIn("1.9-rc-1")),
+
+    META_DATA(ROOT, "metadata",
```

Comment:
I really like how we now keep track of these changes and remove the duplication from our test fixtures 👍